src/axom/quest/SignedDistance.hpp

```diff
@@ -70,14 +70,16 @@ class SignedDistance
    * \param [in] surfaceMesh user-supplied surface mesh.
    * \param [in] isWatertight indicates if the surface mesh is closed.
    * \param [in] maxObjects max number of objects for spatial decomposition.
-   * \param [in] maxLevels max levels for spatial decomposition (optional).
+   * \param [in] maxLevels max levels for spatial decomposition.
+   * \param [in] computeSign indicates if distance queries should compute signs (optional).
    * \note Default maxLevels is 5 if not specified.
```

Comment:
@kennyweiss , I see that the declaration of `SignedDistance()` provides a default value of `true` for argument `computeSign` (new line 82).  Thanks!

I also see the `\note` at (new) line 75 says "Default maxLevels is 5 if not specified."  I don't see this specified as a default in the function declaration.  Where is `maxLevels = 5` specified?

Please add the following line to the Doxygen after (new) line 75:  "\note Default computeSign is true if not specified."

Edit: I see that this comment is in the left column of the GitHub review UI... rather confusing.

Reply:
Fixed.